### ISSUE TITLE
[REQUEST] Installation guide for OpenSUSE Tumbleweed

### ISSUE THREAD

**Issue by Sandaidev** [aristocratos/bashtop#187]
**Is your feature request related to a problem? Please describe.**
No, this isn't a real problem and I'm incredibly ashamed of asking for simple help this way.

**Describe the solution you'd like**
A simple way to install Bashtop on OpenSUSE Tumbleweed.

**Describe alternatives you've considered**
I've considered cloning the repo and installing it manually, though I won't get upstream updates this way.

**Additional context**
I'm a newbie to OpenSUSE, and I don't really know how to use things properly yet, I hope this ticket doesn't break the rules.
Basically, bashtop is not in OpenSUSE Tumbleweed's repositories, running `zypper install bashtop` will result in the package not being found. I don't know if you can or want to do something about it, but I just wanted to let you know about this. Thank you very much.
